components/dashboard/src/settings/Preferences.tsx

```diff
@@ -216,7 +185,27 @@ export default function Preferences() {
                         )}
                         <CheckBox
                             title="Latest Release"
```

Comment:
**suggestion**: What do you think of rephrasing this to include the _Unstable_ term?

> Latest Release (Unstable)

See also relevant discussion in https://github.com/gitpod-io/gitpod/issues/7932#issuecomment-1026672526.